contrib/README.md

```diff
@@ -2,20 +2,29 @@
 
 ### Summary
 
-This project follows [Semantic Versioning 2.0](https://semver.org/). In general, every release is associated with a tag and a changelog. `master` serves as the mainline branch for the project and represent the latest state of development.
+This project follows [Semantic Versioning 2.0](https://semver.org/). In general, every release is associated with a tag and a changelog. `support/3.x` serves as the mainline branch for the project and represent the latest state of development.
 
 ### Publishing a New SDK version
 ```bash
+# fetch and rebase from upstream/support/3.x
+git fetch upstream support/3.x
+git rebase upstream/support/3.x
+
+# checkout a release branch
+git checkout -b release/3.<x>.<y>
+
 # run bump_version.bash with either patch, minor, or major
 ./bump_version.bash <patch|minor|major>
 
-# make live
-git push upstream && git push upstream --tags
-npm publish
+# push changes to your remote origin 
+git push origin release/3.<x>.<y>
 
-npm deprecate mongodb-stitch "The browser SDK has moved to mongodb-stitch-browser-sdk, and the Node.js SDK has moved to mongodb-stitch-server-sdk"
```

Comment:
this `npm deprecate` call still needs to be run after release, so I would just move this after the `npm publish` step

Reply:
woops